app/code/Magento/Store/Model/Store.php

```diff
@@ -1164,11 +1164,12 @@ public function isDefault()
      * Retrieve current url for store
      *
      * @param bool $fromStore
+     * @param bool $clearUrl
      * @return string
      * @SuppressWarnings(PHPMD.CyclomaticComplexity)
      * @SuppressWarnings(PHPMD.NPathComplexity)
      */
-    public function getCurrentUrl($fromStore = true)
+    public function getCurrentUrl($fromStore = true, $clearUrl = false)
```

Comment:
We can not that like that see:
https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/

Adding parameters in public methods 
Deprecate the method and add a new method with the new parameter(s) instead of adding them to a public method.

Follow the alternative implementation described earlier for introducing a new method to a class or interface.
Reference the new method in a @see tag as a recommended replacement. Explain the reasons for replacing the old method with the new one (e.g., there is a bug in the old method).

Reply:
hmm, ok i'm will find a better way , because changing the whole method for a single point is not logical.